src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+                <IssueList id={this.props.login} issues={this.state.data.working} emptyText="Not working on any assigned issues" title="Working on issues" />
+                <IssueList id={this.props.login} issues={this.state.data.prs} emptyText="No pull requests created or assigneds" title="Pull requests" />
+                <IssueList id={this.props.login} issues={this.state.data.other} emptyText="No other assigned issues" title="Other assigned issues" collapsable={true} />
+            </div>
+
+            if (this.props.environment === "Development") {
+                rateLimitDisplay = <span>
+                    <span className="badge" style={headerBadgeStyles}>Rate Limit Cost: {this.state.data.graphQlRateLimit.cost}</span>
+                    <span className="badge" style={headerBadgeStyles}>Page Count: {this.state.data.pages}</span>
+                </span>;
+            }
+        }
+
+        return <div className="panel panel-primary">
+            <div className="panel-heading">
+                <div className="pull-right">
+                    {rateLimitDisplay}
+                    <button className="btn btn-info btn-xs" style={headerBadgeStyles} onClick={() => this.reload()}>
+                        <span className="glyphicon glyphicon-refresh" />
+                    </button>
+                </div>
+                <h3 className="panel-title">
+                    <a href={`https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+assignee%3A${this.props.login}`}>{this.props.login}</a>
+                </h3>
+            </div>
+            {content}
+        </div>;
+    }
+};
+
+interface PageProps {
+    repoSet: string,
+    baseUrl: string,
+    environment: string
+}
+
+interface PageState {
+    loading: boolean,
+    error: string,
+    people: string[],
+    rateLimit: {
+        graphQl: { cost: number, remaining: number, resetAt: string },
+        rest: { remaining: number, reset: string },
+        sinceReload: number
+    }
+}
+
+export class Page extends React.Component<PageProps, PageState> {
+    constructor(props: PageProps) {
+        super(props);
+        this.state = {
+            loading: true,
+            error: "",
+            people: [],
+            rateLimit: {
+                graphQl: null,
+                rest: null,
+                sinceReload: 0
+            }
+        };
+    }
+
+    async componentDidMount() {
+        // Fetch the list of people
+        let newState = { ... this.state };
+        let resp = await fetch(`${this.props.baseUrl}api/repoSets/${this.props.repoSet}/people`, fetchSettings);
+        if (resp.status < 200 || resp.status > 299) {
+            newState.loading = false;
+            newState.error = `Unexpected response from server: ${resp.status} ${resp.statusText}`;
+        }
+        else {
+            newState.loading = false;
+            newState.error = "";
+            newState.people = (await resp.json()) as string[];
```

Comment:
TBH it's a bit unusual to see all these assignments in a React app. Most React devs find that functional programming habits mesh better with React's philosophy. I know it's a minor difference, but it would be more typical to phrase this logic more like:

```
    this.setState({
        loading: false,
        error: '',
        people: await resp.json(), // Not sure you strictly need the `as` cast, if the state type already has a type for `people`
        ...this.state // Instead of line 305
    });
```

This is more just a thought to consider rather than something you should necessarily change, since of course it doesn't actually affect behaviour.